src/CoreShop/Bundle/CoreBundle/Migrations/Version20171220075432.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class Version20171220075432 extends AbstractPimcoreMigration implements ContainerAwareInterface
```

Comment:
Why do we need a migration here? Nothing on shared translations has been changed

Reply:
yea, but some commits before..

Reply:
fair enough :)